src/plugins/workspace/public/components/workspace_creator/workspace_creator.test.tsx

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { WorkspaceCreator } from './workspace_creator';
+import { PublicAppInfo } from 'opensearch-dashboards/public';
+import { fireEvent, render, waitFor } from '@testing-library/react';
+import * as opensearchReactExports from '../../../../opensearch_dashboards_react/public';
+import { BehaviorSubject } from 'rxjs';
+import { WorkspaceFormSubmitData } from './workspace_form';
```

Comment:
It would be better that we import absolute module in the beginning and then relative module. I do not know why OSD lint do not automatically format that but we'd better follow the standard pattern.